packages/e2e/src/specs/help/help-contact-support.e2e.ts

```diff
@@ -0,0 +1,144 @@
+    await expect(attachment).toHaveText(
+      `${year}-${(month > 9 ? "" : "0") + month}-${
+        (day > 9 ? "" : "0") + day
+      }.zip`
+    )
+  })
+
+  it("Click send button and wait for message is sent", async () => {
+    await waitForClickableAndClick(await helpModalPage.sendButton)
+
+    const successModal = await helpModalPage.sentSuccessModal
+    await successModal.waitForDisplayed({ timeout: 90000 })
+    await expect(successModal).toBeDisplayed()
+  })
+
+  it("Close success modal, open new contact support modal and close it, ", async () => {
+    await waitForClickableAndClick(await helpModalPage.closeBottomButton)
+
+    await waitForClickableAndClick(await helpPage.contactSupportButton)
+
+    await waitForClickableAndClick(await helpModalPage.closeModalButton)
+  })
+
+  it("Open contact support modal and try to send it without any user input ", async () => {
+    await waitForClickableAndClick(await helpPage.contactSupportButton)
+
+    await waitForClickableAndClick(await helpModalPage.sendButton)
+
+    await expect(helpModalPage.sendButton).toBeDisabled()
+    await waitForClickableAndClick(await helpModalPage.closeModalButton)
+  })
+
+  after(async () => {
+    try {
+      await waitForClickableAndClick(await helpModalPage.closeModalButton)
+    } catch (error) {
+      console.log(error)
+    }
+    await browser.switchWindow("#/overview")
+  })
+})
+describe("Validate email scenarios on contact support modal", () => {
+  before(async () => {
+    // Waiting for device connected through USB
+    await browser.executeAsync((done) => {
+      setTimeout(done, 10000)
+    })
+    await modalGeneralPage.clickUpdateAvailableModalCloseButton()
+    await waitForClickableAndClick(await navigationTabs.helpTab)
+    await browser.pause(4000)
+    await browser.switchWindow("#/help")
+    await waitForClickableAndClick(await helpPage.contactSupportButton)
+  })
+
+  it("Lack of '@' character", async () => {
+    await insertTextToElement(await helpModalPage.emailInput, "e2emudita.com")
+
+    await expect(await helpModalPage.invalidEmailTextElement.getText()).toEqual(
+      "Email is invalid"
+    )
+  })
+
+  it("Two '@' characters", async () => {
+    await insertTextToElement(await helpModalPage.emailInput, "e2e@@mudita.com")
+
+    await expect(await helpModalPage.invalidEmailTextElement.getText()).toEqual(
+      "Email is invalid"
+    )
+  })
+
+  it("Only '@' and two '..'characters ", async () => {
+    await insertTextToElement(await helpModalPage.emailInput, "@..")
+
+    await expect(await helpModalPage.invalidEmailTextElement.getText()).toEqual(
+      "Email is invalid"
+    )
+  })
+
+  it("Correct email to check Email is invalid text stops being displayed", async () => {
+    await insertTextToElement(
+      await helpModalPage.emailInput,
+      "lukasz@mudita.com"
+    )
+
+    await expect(
+      await helpModalPage.invalidEmailTextElement.getText()
+    ).not.toEqual("Email is invalid")
```

Comment:
Shouldn't we check for exact result instead of checking if the string does not equal something?

Reply:
this is just to check error text is no longer displayed :)